src/main/java/com/timetrackingreminder/TimeTrackingReminderPlugin.java

```diff
@@ -244,6 +251,40 @@ private boolean showSeaweedInfoBox() {
         return prediction.getDoneEstimate() <= unixNow;
     }
 
+    private boolean showSeaweedInfoBox() {
```

Comment:
I think at this point it would be helpful to add a helper function that does the `predictPatch` / `getProduce` / `getCropstate` / `getDoneEstimate` parts, so we can instead loop/call that for logic that has multiple patches.

Maybe `showInfoBoxForPatch(patch, produce)` for the interface.